client/src/components/Tip/Tip.js

```diff
@@ -27,58 +39,40 @@ const tipImportanceMap = {
 };
 
 /**
- * UI element displaying a toggle-able "Tip". Designed to be used as an 'input group suffix'.
+ * UI element displaying a toggle-able "Tip".
+ * THe LABEL type will show next to a form field title label
+ * The INPUT_GROUP type is designed to be used as an 'input group suffix'.
  */
-class Tip extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      open: false,
-    };
-
-    this.handleTipToggle = this.handleTipToggle.bind(this);
+function Tip(props) {
+  const { content, fieldTitle, icon, id, importance } = props;
+  const { iconColor, type } = tipImportanceMap[importance];
+  const label = i18n.inject(i18n._t('Admin.TIP_LABEL', '{type} for {fieldTitle}'), {
+    type,
+    fieldTitle
+  });
+  const classes = ['tip', props.extraClass];
+  if (props.type === TIP_TYPES.TITLE) {
+    classes.push('tip--title');
+  } else if (props.type === TIP_TYPES.INPUT_GROUP) {
+    classes.push('tip--input-group', 'btn--last', 'btn-outline-secondary', `text-${iconColor}`);
   }
+  const buttonId = `${id}-tip`;
+  const buttonProps = {
+    id: buttonId,
+    onClick: () => {},
```

Comment:
What's the point of onClick here?

Reply:
The button component (presumably the reactstrap one) wants an onClick function defined otherwise it doesn't want to work.  So I put a noop in